docker/bacalhau-image/Dockerfile

```diff
@@ -1,10 +1,22 @@
 # syntax=docker/dockerfile:1.4
 # Pinned to 1.20
 FROM cgr.dev/chainguard/go@sha256:8ed3fdc8f6375a3fd84b4b8b696a2366c3a639931aab492d6f92ca917e726ad6 as build
+
+# Release tag. Used to build the binary and tag the version.
+ARG TAG
+
 WORKDIR /work
-COPY Makefile .
+
+# only copy what we need to build the binary instead of
```

Comment:
Not a concern yet, but when we move to pluggable executors we'll probably need to copy the binaries into the container too, right?

Reply:
yeah, and I assume some work is needed to configure the paths of the binaries right?